.gitignore

```diff
@@ -21,3 +21,4 @@ __pycache__
 *.swo
 *~
 htmlcov
+python-env
```

Comment:
⛔ in https://github.com/hellysmile/fake-useragent/pull/136/files means that GitHub UI expects text files to end in one and only on `\n`.

Reply:
True, however it's not the end of the world btw.